util-flywayinstaller/src/main/java/com/github/cafapi/util/flywayinstaller/Migrator.java

```diff
@@ -38,6 +38,7 @@ private Migrator()
     {
     }
 
+    // Overloaded method for backward compatibility
```

Comment:
If this is only being maintained for backward compatibility then put the `@Deprecated` annotation on it so that users are warned that they should stop using it.